master/reference/advanced/etcd-rbac/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Setting up etcd certificates for RBAC
+---
+
+When using etcd it is a good idea to protect the data stored there.  This is
+even more true when you have multiple components using a common etcd cluster.
+
+## Why you might be interested in this guide
+
+- You want to restrict the operations that support staff are able to perform
+  to minimize accidental corruption or deletion of etcd data.
+- You are using the etcd cluster with multiple orchestrators (Kubernetes,
+  docker, Calico) or multiple tenants and want to prevent access between the
+  multiple tenants.
```

Comment:
This bullet is a bit confusing.  Calico is not an orchestrator.  Not clear what tenant means in this context.
I think this bullet should be "You want to use a single etcd cluster for Calico and Kubernetes (rather than having an etcd cluster for Calico and a separate etcd cluster for Kubernetes."